packages/fuel-gauge/src/contract-factory.test.ts

```diff
@@ -7,7 +7,7 @@ import storageSlots from '../fixtures/forc-projects/storage-test-contract/out/de
 
 describe('Contract Factory', () => {
   const createContractFactory = async () => {
-    const provider = new Provider('http://127.0.0.1:4000/graphql');
+    const provider = await Provider.connect('http://127.0.0.1:4000/graphql');
```

Comment:
Should the `FUEL_NETWORK_URL` constant replace this URL?

Several other places are using hardcoded strings.

> This can be another PR though.

Reply:
@arboleya Yeah, this is indeed a problem but one that is prevalent throughout our codebase. I will file a follow-up for this and get this fixed in a separate PR.

Reply:
- https://github.com/FuelLabs/fuels-ts/issues/1258